### ISSUE TITLE
EGNN re-training

### ISSUE THREAD

**Issue by Dornavineeth** [guanjq/targetdiff#10]
Hi @guanjq , I am not able to replicate the results by re-training the EGNN model . However While using the pre-trained model I am able to replicate the results of `RMSE: 1.316, MAE: 1.031, R^2 score: 0.633, Pearson: 0.797, Spearman: 0.782, mean/std: 6.412/1.621`.

I tried to keep the all the hyper-parameters and datasets(`/data/pdbbind_v2016/pocket_10_refined`) by referring  to the config found in the checkpoint shared and followed the readme to prepare the pockets and splits

My current results on the test set shared are
`RMSE: 3.082, MAE: 2.412, R^2 score: -1.014, Pearson: 0.513, Spearman: 0.562, mean/std: 7.769/3.195`

Any Idea what might me going wrong in re-training?

_Originally posted by @Dornavineeth in https://github.com/guanjq/targetdiff/issues/1#issuecomment-1589863780_

**Comment by Dornavineeth** [comment]
Hi @guanjq , I just figured out the there is a minor bug in the [train_prop.py](https://github.com/guanjq/targetdiff/blob/da0b8d87821ec0a9f74b5df0d50b4c9929dfdba9/scripts/property_prediction/train_prop.py#L96). The `break` statement is making the model to train only on one batch in an epoch.

**Comment by guanjq** [comment]
Hi @Dornavineeth , thank you for letting me know! I add the `break` statement when testing the code before making it public, but forgot to remove it... I have fixed it. Thank you very much!
